gravitee-apim-console-webui/src/entities/management-api-v2/documentation/createDocumentation.ts

```diff
@@ -36,4 +38,18 @@ export interface CreateDocumentationMarkdown extends BaseCreateDocumentation {
   homepage?: boolean;
 }
 
-export type CreateDocumentation = ({ type: 'FOLDER' } & CreateDocumentationFolder) | ({ type: 'MARKDOWN' } & CreateDocumentationMarkdown);
+export interface CreateDocumentationSwagger extends BaseCreateDocumentation {
+  content?: string;
+  homepage?: boolean;
+}
+
+export interface CreateDocumentationAsyncApi extends BaseCreateDocumentation {
+  content?: string;
+  homepage?: boolean;
+}
```

Comment:
Are those types exclusives on purpose or could they be merged along with the `CreateDocumentationMarkdown` one under a `CreateDocumentationFile` interface ?

Reply:
I would keep them exclusive to mimic how the objects are constructed in the back-end `openapi` file.